pom.xml

```diff
@@ -215,4 +227,54 @@
 	</distributionManagement>
 	
 
+	<!-- different profiles for building/testing against different version of 
+		openmrs -->
+	<profiles>
+		<profile>
+			<id>openmrs-1.9</id>
+
+			<properties>
+				<openMRSVersion>1.9.9</openMRSVersion>
+				<openMRSMinorVersion>1.9</openMRSMinorVersion>
+				<webservicesRestVersion>2.13-SNAPSHOT</webservicesRestVersion>
```

Comment:
Why would we test against different versions of the REST module? I would prefer that we just make this module require the latest (now snapshot) version of REST, and make that common in both profiles...